indico/modules/users/templates/dashboard.html

```diff
@@ -167,80 +181,43 @@
-                    } else {
-                        var now = moment();
-                        var past_events = _.filter(resp.results, function(e) {
-                            return momentDate(e.endDate) < now;
-                        });
-                        var current_events = _.filter(resp.results, function(e) {
-                            return momentDate(e.startDate) < now && now < momentDate(e.endDate);
-                        });
-                        var future_events = _.filter(resp.results, function(e){
-                            return now < momentDate(e.startDate);
-                        });
-                        resp.results = past_events.concat(current_events, future_events);
-                        $.each(resp.results, function (i, item) {
-                            html += '<li id="event-' + item.id + '" class="truncate"> \
-                                     <a href="' + item.url + '" class="truncate"> \
-                                         <span class="event-date">' + getDate(item.startDate, item.endDate) + '</span> \
-                                         <span class="event-title truncate-target">' + item.title + '</span> \
-                                         <span class="item-legend"> \
-                                             <span title="' + $T("You have management rights") + '" class="icon-medal contextHelp ' + hasRights(item.roles, managementFilter) + '"></span> \
-                                             <span title="' + $T("You are a reviewer") + '" class="icon-user-reading contextHelp ' + hasRights(item.roles, reviewFilter) + '"></span> \
-                                             <span title="' + $T("You are an attendee") + '" class="icon-ticket contextHelp ' + hasRights(item.roles, attendanceFilter) + '"></span> \
-                                         </span> \
-                                     </a>\
-                                 </li>';
-                        });
-                    }
-                    var list = $("#yourEvents ol");
-                    list.append(html);
-                    // Enable tooltips for active roles and delete them for inactive ones
-                    list.find('.contextHelp[title].active').qtip();
-                    list.find('.contextHelp[title]:not(.active)').removeAttr('title');
-                });
+            {% if linked_events %}
+                var linkedEvents = {{ linked_events|tojson }};
+                var list = $('#yourEvents ol');
+                list.append(_.map(linkedEvents, function(event) {
+                    return '<li id="event-' + event.id + '" class="truncate"> \
+                            <a href="' + event.url + '" class="truncate"> \
+                                <span class="event-date">' + getDate(event.start_dt, event.end_dt) + '</span> \
```

Comment:
I kept the JS rendering of the data to avoid having to rewrite the "calendar"-style date formatting in Python (as `format_human_date` doesn't do exactly the same thing). I guess at some point this should be done though...